backend/src/main/java/com/jobwait/domain/Answer.java

```diff
@@ -0,0 +1,27 @@
+package com.jobwait.domain;
+
+public class Answer<T> {
+    private T answerValue;
+    private String answerType;
+
+    public Answer(T answerValue) {
+        this.answerValue = answerValue;
+    }
+
+    public Answer(T answerValue, String answerType) {
+        this.answerValue = answerValue;
+        this.answerType = answerType;
+    }
+
+    public T getValue() {
+        return this.answerValue;
+    }
+
+    public void setValue(T value) {
+        this.answerValue = value;
+    }
+
+    public String getAnswerType() {
+        return this.answerType;
+    }
+}
```

Comment:
look into the `record` type in java, there's an example of it somewhere in the existing code. you can get rid of a lot of this boilerplate

Reply:
it makes it quite annoying to setup multiple constructors of varying types so i chose to pass on it, its the same reason i removed it from Answers